lib/public/components/Log/logCollapsed.js

```diff
@@ -62,32 +83,35 @@ const textBox = ({ text, id }, onClick) =>
  *
  * @param {Log} log the log to display
  * @param {boolean} highlight indicator if this log should be highlighted
+ * @param {boolean} showLogTitle indicator if the title of this log should be visible
  * @param {function} onExpand function called when the log should be expanded
  * @return {Component} the log's display component
  */
-export const logCollapsed = (log, highlight, onExpand) => {
+export const logCollapsed = (log, highlight, showLogTitle, onExpand) => {
     const logFields = activeFields();
-    // eslint-disable-next-line require-jsdoc
 
-    return h(`#log-${log.id}.w-100.flex-row.m1.shadow-level1`, [
-        h(`.w-20.p2.shadow-level1${highlight ? '.bg-gray-light' : ''}`, [
-            h('table', [
-                h('tbody', Object.entries(logFields)
-                    .map(([
-                        key, {
-                            name,
-                            format,
-                            visible,
-                        },
-                    ]) => visible && h(`tr#log-${log.id}-${key}`, [
-                        h('td.text-right', { style: 'font-weight: bold' }, `${name}:`),
-                        h('td', format ? format(log[key]) : log[key]),
-                    ]))),
+    // eslint-disable-next-line require-jsdoc
+    return h(
+        `#log-${log.id}.w-100.m1.p3.shadow-level1${highlight ? '.b1.b-primary' : ''}`,
+        h(
+            'div.flex-wrap.justify-between.mv1',
+            showLogTitle ? h(`h4#log-${log.id}-title`, log.title) : '',
+            h('div.flex-row.justify-between.w-100', [
```

Comment:
I'm pretty sure you can do without w-100 (this class should be avoided) using flex grow for example (I just see that it's in the framework example... which is bad)